scenarios.js

```diff
@@ -1,25 +1,29 @@
 SCENARIO_DEFINITIONS =
-    [   { name: "#1 Black Barrow"
+    [   { number: 1

```

Comment:
As mentioned above - we could avoid the new `number` property and rely on array index instead so long as we account for the offset of 1.